### ISSUE TITLE
inViewport.startIntersectionObserver is not a function

### ISSUE THREAD

**Issue by edeis53** [DockYard/ember-in-viewport#201]
I was actually just preparing a demo repo to reproduce another bug for iOS and found this.

Latest version of in-viewport fails. Both in fresh app and upgrading in our existing app.

## Version

"ember-in-viewport": "^3.5.8"
"ember-cli": "~3.7.1",

## Test Case

Use this repo at this revision:
https://github.com/edeis53/inviewport-ios-issue/commit/604265ae5a8a325b44fee27bf4df35ef3d8a2912

## Steps to reproduce

`ember server` then navigate to http://localhost:4200/h-scroll-window

## Expected Behavior

viewport detection works.

## Actual Behavior

inViewport service doesn't start,
```
ncaught TypeError: inViewport.startIntersectionObserver is not a function
    at Class._setInitialViewport (in-viewport.js:121)
    at Class.watchElement (in-viewport.js:79)
    at Class.didInsertElement (in-viewport.js:60)
    at Class.superWrapper [as didInsertElement] (utils.js:286)
    at Class.trigger (core_view.js:69)
    at Class.superWrapper [as trigger] (utils.js:286)
    at CurlyComponentManager.didCreate (glimmer.js:3874)
    at Transaction.commit (runtime.js:2678)
    at Environment$1.commit (runtime.js:2766)
    at Environment$1.commit (glimmer.js:3031)
```

**Comment by edeis53** [comment]
actually, going to close for now. I think I have an idea on what the issue is.

**Comment by snewcomer** [comment]
hi @edeis53.  what was the issue?

**Comment by edeis53** [comment]
We had a component with the property `inViewport` that was consuming the `InViewportMixin`. We got the error after upgrading to the latest version of ember-in-viewport, which I can see injects a service as `inViewport`. I renamed the component's property to something else and all is good there.

Now I have a full working test case for a much better bug submitted! https://github.com/DockYard/ember-in-viewport/issues/202
